packages/cdk8s-cli/bin/cmds/init.ts

```diff
@@ -2,6 +2,8 @@ import * as yargs from 'yargs';
 import * as fs from 'fs-extra';
 import * as path from 'path';
 import { sscaff } from 'sscaff';
+const constructsVersion = require('../../package.json').dependencies.constructs;
+const jsiiRuntimeVersion = '1.5.0'
```

Comment:
I am wondering if we really need `jsii` as a direct dependency. It will be brought in by `constructs` and by `cdk8s` as a transitive dependency, wouldn't that be easier?